scripts/ddev-wsl2-postinstall.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Post-install script for ddev-wsl2 package
+# Unblocks Windows executables to prevent security warnings
+
+echo "Configuring WSL2 security settings..."
+
+# Try to find and use reg.exe
+for REG_EXE in /mnt/*/Windows/System32/reg.exe; do
+    echo "Attempting registry modification via reg.exe..."
+    REG_OUTPUT=$("$REG_EXE" add "HKCU\\Software\\Microsoft\\Windows\\CurrentVersion\\Internet Settings\\ZoneMap\\Domains\\wsl.localhost" /v "file" /t REG_DWORD /d 1 /f 2>&1)
+    REG_EXIT_CODE=$?
+    if [ $REG_EXIT_CODE -eq 0 ]; then
+        echo "WSL2 security settings configured successfully via registry"
+        exit 0
+    else
+        echo "Registry method failed: $REG_OUTPUT"
+    fi
+    break  # Only try the first one found
+done
+
+# If reg.exe method failed or not found, show manual instructions
+echo "Note: Could not automatically configure WSL2 security settings."
+echo "To resolve Windows security warnings manually:"
+echo "1. Open Internet Options (Control Panel > Internet Options)"
+echo "2. Go to Security tab > Local Intranet > Sites > Advanced"
+echo "3. Add this website to the zone:"
+echo "   - \\\\wsl.localhost"
```

Comment:
I see this is added by the package:

<img width="377" height="341" alt="img" src="https://github.com/user-attachments/assets/abf76224-74b8-45bb-81cd-5ee6ccfb12d5" />

But:

```
Note: Could not automatically configure WSL2 security settings.
To resolve Windows security warnings manually:
1. Open Internet Options (Control Panel > Internet Options)
2. Go to Security tab > Local Intranet > Sites > Advanced
3. Add this website to the zone:
   - \wsl.localhost
```

`\wsl.localhost` is not a valid website.

Reply:
It seems that it should be suggesting `\\wsl.localhost` with the right number of escapes. `wsl.localhost` doesn't seem to come out the same. The correct result (after closing the dialog and reopening it) is `file://*.wsl.localhost`. This is pretty weird stuff.

Reply:
I copied that `echo` in my shell on Linux without realizing it's zsh, not bash.